cmd/api/app/server.go

```diff
@@ -66,6 +67,13 @@ func newServerCommand() *cli.Command {
 			if err != nil {
 				return err
 			}
+			if srvr.Config.Facility.TrafficAlerts.Enabled {
```

Comment:
I didn't see a pattern for this, but I did see config features with enabled/disabled flags.